create_book.rb

```diff
@@ -12,11 +14,40 @@ def create_book
     book_name = gets.chomp
     puts 'Enter book Author:'
     book_author = gets.chomp
+
+    # Load existing books from the JSON file
+    existing_books = load_books_from_json
+
     added_book = Book.new(book_name, book_author)
     @books.push(added_book)
+    existing_books.push({ title: added_book.title, author: added_book.author })
+
+    save_books_to_json(existing_books)
```

Comment:
- [ ] Hey Team! 👋 Stellar job on implementing the method for saving books to JSON. 📚 However, there's a slight deviation from the project requirement. 🚧 According to the specifications, we're supposed to save to JSON only on app exit, not immediately upon book creation. 🔄 Kindly review the current implementation and make the necessary adjustments to align with the project's directive. Let's ensure we save that data only when the app bids us farewell. 🌟 Thanks for your attention to detail! 🙌